style.css

```
@@ -931,8 +832,9 @@ a.reddit-join-button:hover {
     left: 0;
     right: 0;
     bottom: 0;
-    background: radial-gradient(ellipse at top left, rgba(79, 70, 229, 0.18) 0%, rgba(16, 185, 129, 0.08) 45%, transparent 75%),
-                radial-gradient(ellipse at bottom right, rgba(14, 165, 233, 0.16) 0%, transparent 65%);
+    background: rgba(255, 255, 255, 0.45);
+    background: radial-gradient(ellipse at top left, rgba(247, 254, 249, 0.35) 0%, rgba(255, 255, 255, 0.3) 50%),
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)